attributes/default.rb

```diff
@@ -12,10 +12,10 @@
 default['pgd']['static_url'] = "/static/"	#Trailing slash is must
 
 # Settings.py database
-default['pgd']['database']['engine'] = "django.db.backends.mysql"
-default['pgd']['database']['name'] = "pgd-dev"
-default['pgd']['database']['user'] = "pgd-user"
-default['pgd']['database']['password'] = "pdg-pass"
+default['pgd']['database']['engine'] = nil
```

Comment:
@jordane What would be thest best way to document this? Comment in the default.rb file or a ntoe in the readme?